### DISCUSSION TITLE
Link to Alert message and title template documentation

### DISCUSSION THREAD

**Discussion by joeblubaugh** [grafana/grafana#49862] [Alerting]
Grafana Alerting supports templating the alert message and, for [certain notifiers, the alert subject](https://github.com/grafana/grafana/pull/49742). Finding the [documentation for the supported template variables](https://grafana.com/docs/grafana/latest/alerting/unified-alerting/message-templating/template-data/) requires that you actively search the docs, opening a separate window. We should reduce the friction to take advantage of message templating.
I'm requesting that we link to the alerting template documentation from every field in the product that can be templated - though I think exploring an in-product documentation approach could also be interesting.

cc @jessover9000 @grafana/alerting-squad-frontend @JacobsonMT

**Comment by joeblubaugh** [comment]
We also have a plan to standardize support for templating across contact points: https://github.com/grafana/grafana/issues/49716

**Comment by jessover9000** [comment]
For now, imiproving the docs links sounds like a great first step that should be quick to implement! There is a kickoff call to start planning improvements for message templating, and I totally agree that an overhaul there should include better in-app documentation.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
